### PR TITLE
Update mod.yaml to use the new package notation

### LINKED ISSUE #117: Crash (File not found: ra2:uibits/loadscreen.png)
...
Loading mod: ra2
Exception of type `System.IO.FileNotFoundException`: File not found: ra2:uibits/loadscreen.png
  at OpenRA.FileSystem.FileSystem.Open (System.String filename) [0x00000] in <filename unknown>:0 
  at OpenRA.Mods.Common.LoadScreens.LogoStripeLoadScreen.Init (OpenRA.ModData modData, System.Collections.Generic.Dictionary`2 info) [0x00000] in <filename unknown>:0 
  at OpenRA.ModData..ctor (System.String mod, Boolean useLoadScreen) [0x00000] in <filename unknown>:0 
  at OpenRA.Game.InitializeMod (System.String mod, OpenRA.Arguments args) [0x00000] in <filename unknown>:0 
  at OpenRA.Game.Initialize (OpenRA.Arguments args) [0x00000] in <filename unknown>:0 
  at OpenRA.Program.Run (System.String[] args) [0x00000] in <filename unknown>:0 
  at OpenRA.Program.Main (System.String[] args) [0x00000] in <filename unknown>:0 
AL lib: (EE) alc_cleanup: 1 device not closed
...
...

This Should fix it? 

In /mods/ra2/mod.yaml

Image: ra2:uibits/loadscreen.png ->>>> Image: ./mods/ra2/uibits/loadscreen.png

### PR DESCRIPTION
**Pull request by abcdefg30**
Depends on https://github.com/OpenRA/OpenRA/pull/10703.
Fixes #117.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Updated and rebased as OpenRA/OpenRA#10703 was merged.

**Comment by abcdefg30** [comment]
Merging with the :+1: of @Phrohdoh.

File: mod.yaml [hunk 1]
```diff
@@ -198,32 +202,32 @@ LobbyDefaults:
 	TechLevel: Unrestricted
 
 ChromeMetrics:
-	ra2:metrics.yaml
+	ra2|metrics.yaml
```
Comment (GraionDilach) [inline mod.yaml]: Why don't we use the recently added common metrics as well?

Comment (abcdefg30) [inline mod.yaml]: I think that can be done in another pr (obviously has to, as this one is merged :p).

Comment (GraionDilach) [inline mod.yaml]: Yeah, I guess so, I just only noticed it now. :P

**Pull request merged**